README.md

```diff
@@ -71,6 +71,18 @@ then to point Halide to it:
 
 (Note that you *must* add `clang` to `LLVM_ENABLE_PROJECTS`; adding `lld` to `LLVM_ENABLE_PROJECTS` is only required when using WebAssembly, but we recommend enabling it in all cases, to simplify builds.)
 
+#### Building Halide - Using vcpkg
```

Comment:
Can we nest this under "Building Halide and LLVM on Windows" below? Windows is a minority of our users, so it would be strange to put it above make and cmake.

I will test the instructions on my Windows machine tonight. I haven't used vcpkg there before so it should be a good test of a clean install.

Reply:
Yes, I changed it.